src/pages/groupComparison/Overlap.tsx

```diff
@@ -57,7 +59,29 @@ export default class Overlap extends React.Component<IOverlapProps, {}> {
 
     @autobind
     private getSvg() {
-        return document.getElementById(SVG_ID) as SVGElement | null;
+        if (this.plotType.result! === PlotType.Upset) {
```

Comment:
not sure what all this is about.  why have we left React paradigm?  there some stuff i would improve about the below but i want to first understand the bigger picture.

Reply:
@alisman do you mean instead of using a ref?

Reply:
ok, i see.  ok, lets switch to use ref paradigm.  i know it's an "academic" change but i think it's important to avoid anti-patterns that might screw things up when other devs go at it.

Reply:
you could use react-dom library to find a components top level svg element

Reply:
```node = ReactDOM.findDOMNode(this);``` where this = instance of component

Reply:
thats a great replacement